openedx/core/lib/api/paginators.py

```diff
@@ -26,15 +25,17 @@ def paginate_search_results(object_class, search_results, page_size, page):
         if page == 'last':
             page_number = paginator.num_pages
         else:
-            raise Http404(_("Page is not 'last', nor can it be converted to an int."))
+            raise Http404("Page is not 'last', nor can it be converted to an int.")
 
     try:
         paged_results = paginator.page(page_number)
-    except InvalidPage as e:  # pylint: disable=invalid-name
-        raise Http404(_('Invalid page (%(page_number)s): %(message)s') % {
-            'page_number': page_number,
-            'message': str(e)
-        })
+    except InvalidPage as exception:
```

Comment:
Just FYI our pylint rules are such that variables need to be 3 letters or longer.